_config.yml

```diff
@@ -1,38 +1,47 @@
 # Build settings
 permalink: /blog/:year/:title/
 markdown: kramdown
-highlighter: pygments
+highlighter: rouge
 plugins: [jekyll-paginate-v2]
 
 pagination:
-  enabled: false
+    enabled: false
 
 comments: true
 
 #source: source
 destination: _site
 sass:
-  sass_dir: _sass
-  style: compressed # expanded
+    sass_dir: _sass
+    style: compressed # expanded
 
 # SEO settings
 title: UCD Netsoc
-description: "UCD's Computing and Networking Society"
+description: "UCD's Networking and Computing Society"
```

Comment:
Can you change this to UCD's Internet & Computer Science Society? (I know, we're really inconsistent about this)